### PR TITLE
migrated to @roxi/routify

### PR DESCRIPTION
**Pull request by hobbitronics**

### CODE REVIEW

File: .routify/routes.js [hunk 1]
```diff
@@ -0,0 +1,399 @@
+
```
Comment (longrunningprocess) [inline .routify/routes.js]: are you certain this one should be checked in?

**Review by longrunningprocess — CHANGES_REQUESTED** [review]
the `sveltech` to `@roxi` makes sense but some of the `.routify/*` files are either not needed or an unfortunate configuration now necessary...

File: .routify/routes.js (hunk 1 above)
Comment (hobbitronics) [reply .routify/routes.js]: When I build it generates this so maybe add this to .gitignore? The migration guide gives two options:
routifyDir has been defaulted to .routify.
Option A) Update to import { routes } from '../.routify/routes' in App.svelte
Option B) Set routifyDir to node_modules/@roxi/routify/tmp/routes.

File: rollup.config.js [hunk 2]
```diff
@@ -19,10 +19,11 @@ export default {
 		format: 'iife',
 		sourcemap: production,
 	},
+	inlineDynamicImports: true,
```
Comment (hobbitronics) [inline rollup.config.js]: instead of this we could add
  "routify": {
    "dynamicImports": false
  }
to the package.json. Not sure which is preferable.

Comment (hobbitronics) [reply rollup.config.js]: I decided to go with the suggestion made in the issue [here](https://github.com/roxiness/routify.dev/issues/218)

**Review by longrunningprocess — APPROVED** [review]
this is much better

**Review by briskt — APPROVED** [review]

**Pull request merged**